tools/tplg_parser/tplg_parser.c

```diff
@@ -60,6 +60,36 @@ static enum sof_comp_type find_process_comp_type(enum sof_ipc_process_type type)
 	return SOF_COMP_NONE;
 }
 
+static int is_valid_priv_size(size_t old_size, size_t priv_size,
```

Comment:
return bool maybe?

Reply:
and the first argument could have more intuitive name like size_read maybe?

Reply:
Yeah. Those are better choices. I will fix this.